src/store/modules/web3.ts

```diff
@@ -100,9 +102,11 @@ const actions = {
       } catch (e) {
         console.error(e);
       }
+      const profile = await Box.getProfile(account);
```

Comment:
This can be loaded in the same time than `lookupAddress` on line 101 using `Promise.all()`. That would make login resolve faster.